plugins/thermodyn/src/rendering/BoxRenderer.cpp

```diff
@@ -132,11 +147,16 @@ bool megamol::thermodyn::rendering::BoxRenderer::GetExtents(core::view::CallRend
     BoxDataCall* inBoxCall = nullptr;
     core::moldyn::MultiParticleDataCall* inParCall = nullptr;
     if ((inBoxCall = dataInSlot_.CallAs<BoxDataCall>()) != nullptr) {
-        if (!(*inBoxCall)(0)) return false;
+        if (!(*inBoxCall)(1)) return false;
+        call.AccessBoundingBoxes().SetBoundingBox(inBoxCall->AccessBoundingBoxes().ObjectSpaceBBox());
+        call.AccessBoundingBoxes().SetClipBox(inBoxCall->AccessBoundingBoxes().ObjectSpaceClipBox());
 
+        // this old code calculates the bounding box from the data instead of just reading it
+        /*
+        if (!(*inBoxCall)(0)) return false;
         auto const boxes = inBoxCall->GetBoxes();
 
-        if (boxes->empty()) {
+        if (boxes == nullptr || boxes->empty()) {
             call.AccessBoundingBoxes().SetBoundingBox({-1.0f, -1.0f, -1.0f, 1.0f, 1.0f, 1.0f});
             call.AccessBoundingBoxes().SetClipBox({-1.0f, -1.0f, -1.0f, 1.0f, 1.0f, 1.0f});
         } else {
```

Comment:
Please remove unused code.

Reply:
I reintroduced the old commented out code. It is now available by switching on a bool gui parameter.